### ISSUE TITLE
spt quits updating the playing window at the bottom and you have to 'q' out and restart to fix ipso facto

### ISSUE THREAD

**Issue by JesusChristnt** [Rigellute/spotify-tui#448]
I have spt running on FreeBSD version 11.3 and it works enough to thoroughly pleased with it but the player becoming unresponsive after a few hours of playing is true enough that I have had to 'q' out, at least that is responsive, several times since I installed it. This is probably just a p3 or worse error so handle as you need with sort. I use a urgent and important sort for my work try that instead of just urgent sorting. Thanks for writing spt, it would be sad to have only Radio Paradise which web plays for music but I like spotify for it's great library and playlist making. Having both choices is good enough.

**Comment by trames57** [comment]
Yeah, this has been happening for a while (for me). It used to completely freeze spt, but that has been fixed and is responsive to the "q" now. If I press the "a", I get the general Spotify error. I Ctrl-C out of spt, go back in and press "a" which restores the currently playing track (using spotifyd, so the music never stops playing).

This always happens at the end of a track. The timer bar at the bottom is at 100% played.  I don't see any pattern as to when it may happen meaning it can be less than a few hours of playing. I have it happen within 10-15 minutes.

Arch Linux
spotify-tui 0.18.0
spotifyd  0.2.24
using kitty terminal emulator (as was suggested to me from the "freezing" issue)
